### ISSUE TITLE
Feature request - Ability to change icon sizes in windows-list panel applet.

### ISSUE THREAD

**Issue by lukolszewski** [mate-desktop/mate-panel#1329]
This is a feature request.

On high resolution screens 16x16 mini icons used by windows-list applet are too small. Other applets such as menu can have their icon sizes changed, but windows-list doesn't. This makes rapid selection of a window to switch to impossible on high dpi monitors. No setting/theme etc makes any difference to windows-list icon sizes. The defaults seem to be hardcoded in libwnck and the windows-list applet itself doesn't change the sizes at all.

**Comment by muktupavels** [comment]
Wrong, icon sizes are not hardcoded! There is `wnck_set_default_mini_icon_size` and `wnck_set_default_icon_size`.
